### DISCUSSION TITLE
Tab groups disabled by default since 93.0.4577.83

### DISCUSSION THREAD

**Discussion by sphinxcdi** [bromite/bromite#1419] [General]
Since [93.0.4577.83 disables tab groups](https://github.com/bromite/bromite/blob/master/CHANGELOG.md#930457783) by default, issues like https://github.com/bromite/bromite/issues/1417 / https://github.com/bromite/bromite/issues/1418 are expected.  
I'm creating this to let @salmanpy444 and others know that it can be enabled again by going to `chrome://flags` and explicitly setting `#enable-tab-groups` and `#enable-tab-groups-ui-improvements` to `Enabled`. Doing it both before and after updating has been confirmed to go well by myself and another user.

Thanks for the update.

**Comment by csagan5** [comment]
You can find all changes in the CHANGELOG.md; it is best to read that before submitting issues.

**Reply by docgunthrop** [reply]
A little late to the discussion, but I'm curious what the rationale was for disabling it by default. I looked at the two closed issues regarding this change and there wasn't any reasoning provided. It appears to be on by default for the Android Chromium forks that I've come across.

I can see why tab groups may be useful for a user, but I don't see any downsides to it.

**Reply by suicideboy98** [reply]
i agree

**Comment by suicideboy98** [comment]
thanks i was wondering about this too

**Comment by ronso0** [comment]
Just updated from 91 to 106 and

> going to `chrome://flags` and explicitly setting `#enable-tab-groups` to `Enabled`

alone didn't do anything, still had 60+ single tabs floating around.
In `chrome://flags` I searched for groups and set the other group options to `Enabled`, can't tell which one finally did the trick (and I'm not inclined to test it now since I don't want to lose groups again):
`#enable-tab-groups-continuation`
`#enable-tab-groups-ui-improvements`

**Reply by csagan5** [reply]
`#enable-tab-groups` and `#enable-tab-groups-ui-improvements` both need to be enabled to enable the tab groups functionality.

**Reply by ronso0** [reply]
:+1: 
FWIW, the first time I used the tab overview after updating and re-enabling the group functionality the tab preview images were wrong (so I accidentally merged two tab groups :| )
After that everything was back to normal.
